app/assets/javascripts/controllers/cloud_network/cloud_network_form_controller.js

```diff
@@ -26,6 +27,12 @@ ManageIQ.angular.app.controller('cloudNetworkFormController', ['cloudNetworkForm
       vm.cloudNetworkModel.vlan_transparent = false;
       vm.afterGet = true;
       vm.modelCopy = angular.copy( vm.cloudNetworkModel );
+
+      API.get('/api/providers?collection_class=ManageIQ::Providers::NetworkManager&attributes=id,name&expand=resources')
```

Comment:
I believe this API.get code is repeated many times and could go to  app/assets/javascripts/services/miq_service.js or equivalent (DRY).